packages/wpcom-checkout/src/payment-methods/p24.tsx

```diff
@@ -31,23 +57,23 @@ export function createP24PaymentMethodStore() {
 		},
 	};
 
-	const selectors = {
+	const selectors: StoreSelectorsWithState< NounsInStore > = {
 		getCustomerName( state ) {
-			return state.customerName || '';
+			return state.customerName;
 		},
 		getCustomerEmail( state ) {
-			return state.customerEmail || '';
+			return state.customerEmail;
 		},
 	};
 
 	const store = registerStore( 'p24', {
```

Comment:
I'm wondering if the typing on this function call could help with a lot of the other type assignment being done in this file.

Reply:
Like what?

Reply:
Either they could be inferred from what's passed to the function or explicitly defined in the generic args? (which should be identical syntactically). The object spread in the returned object is what makes it really feel like there's some typing missing—it feels like that object should be able to be returned directly instead of having the additional type casting.